README.md

```diff
@@ -114,3 +114,45 @@ The interesting fields for this use case are:
   the SSO user ID to be sent to Segment
 - `status.compliantUsername` - Contains the username used in the cluster audit
   log.
+
+## Setting up a containerized Splunk instance
+- Either podman or docker can be user
```

Comment:
```suggestion
- Either podman or docker can be used
```

Reply:
It's a mystery how I didn't notice all the leftover duplicated text after re-reading it 3  times... fixed.